### ISSUE TITLE
Feature request: Rotate photo

### ISSUE THREAD

**Issue by ScrumpyJack** [LycheeOrg/Lychee-v3#315]
Would be nice to have some basic image editing like rotating a photo once uploaded

**Comment by ildyria** [comment]
Hummm,
I am not really in favor of that for 2 reasons.
1. this is opening the door to other requests like cropping...
2. this would cluster the UI even more.

**Comment by expiredemulsion** [comment]
I believe that image editing and manipulation should be done before Lychee.
Lychee should keep its idea of easy-to-use, manage and share photos with.

**Comment by ScrumpyJack** [comment]
Ok, good points, thanks for the response. Closing
